plugins/lacrosse-scoreboard/game_renderer.py

```
@@ -464,47 +469,105 @@ def _draw_live_game_status(self, draw: ImageDraw.Draw, game: Dict) -> None:
+    def _draw_dynamic_odds(
+        self,
+        draw: ImageDraw.Draw,
+        odds: Dict[str, Any],
+        width: int,
+        height: int,
+    ) -> None:
+        """Draw odds with dynamic positioning — spread on favored team's side, O/U on opposite."""
+        try:
+            if not odds or isinstance(odds, dict) and any(
+                isinstance(v, type) and hasattr(v, "__call__") for v in odds.values()
+            ):
```

Ruff (0.15.10)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)